index.html

```diff
@@ -15,43 +15,55 @@
 </head>
 
 <body>
-  SpaceWalker
-
-  Home
-  About Us
-  Blog
-  Contact
-  Sign In
-
-  <img src="img/main-header.jpg" alt="Space view of Earth">
-
-  Lift Off
-
-  As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.
-
-  <img src="img/blast.jpg" alt="Rocket blasting off">
-
-  <img src="img/sky.jpg" alt="Space sky image">
-
-  <img src="img/launch.jpg" alt="Launching rocket">
-
-  Can We Visit Planets?
-
-  Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.
-
-  which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.
-
-  Let's Go For A Walk
-
-  The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.
-
-  It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.
-
-  Sign Up
-
-  <img src="img/space-walk.jpg" alt="Space walking astronaut">
-
-  Copyright SpaceWalkers 2018
-
+<div class="home">
+    <header class="main-nav">
+        <h1 class="logo-heading">SpaceWalker</h1>
+
+        <nav>
+            <a class="nav-link" href="index.html">Home</a>
+            <a class="nav-link" href="">About Us</a>
+            <a class="nav-link" href="">Blog</a>
+            <a class="nav-link" href="">Contact</a>
+            <a class="nav-link" href="">Sign In</a>
+        </nav>
+  </header>
+  <img class="banner" src="img/main-header.jpg" alt="Space view of Earth">
+
+    <section class="top-content">
+        <h2>Lift Off</h2>
+
+        <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+        <div class="images-bar">
+            <img src="img/blast.jpg" alt="Rocket blasting off">
```

Comment:
Think about applying unique class names to each image. That may set you up better when styling them.